src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -14,6 +14,8 @@ import java.time.Instant
 class ComposeUp extends DefaultTask {
 
     private Map<String, ServiceInfo> servicesInfos = new HashMap<>()
+    private List<String> composeFiles

```

Comment:
I use this local copy for the case where there is a `docker-compose.override.yml`. In this case a plain `docker-compose up -d` (without -f), but then both yaml's are added to `composeFiles` so service info collection does include the override.
While `composeFiles` changes the `extension.useComposeFiles` does not. It reflects the user's preference.